packages/deprecated/README.md

```diff
@@ -68,7 +68,7 @@ message is only logged once.
 
 _Type_
 
--   `Object` 
+-   `Record<string,(|undefined)>` 
```

Comment:
This wasn't picked up properly.

Reply:
Interesting, is it logged somewhere?

Reply:
It is now 🙂 

https://github.com/WordPress/gutenberg/issues/26971